packages/next/src/cli/next-info.ts

```diff
@@ -104,6 +108,35 @@ async function printDefaultInfo() {
   const installedRelease = getPackageVersion('next')
   const nextConfig = await getNextConfig()
 
+  let staleness = ''
+  let title = ''
+  let versionInfo
```

Comment:
Is `staleness` different than `versionInfo.staleness`? Looks like it so perhaps it should have a different name.